docs/MigrationGuide.md

```diff
@@ -18,11 +20,11 @@ To see the biggest differences please consult the [changelog](Changelog.md).
 * [#3029](https://github.com/TouK/nussknacker/pull/3029) `KafkaConfig` has new field `schemaRegistryCacheConfig: SchemaRegistryCacheConfig`.
 * [#3040](https://github.com/TouK/nussknacker/pull/3040) Deprecated `pl.touk.nussknacker.engine.api.ProcessListener.sinkInvoked` method. Switch to more general `endEncountered` method.
 * [#3076](https://github.com/TouK/nussknacker/pull/3076) new implicit parameter `componentUseCase: ComponentUseCase` was added to `invoke()` method of all services extending `EagerServiceWithStaticParameters`  
-
+* [#2992](https://github.com/TouK/nussknacker/pull/2992) OnDeployActionSuccess in ProcessChangeEvent now requires instance of Option[Comment] instead of Option[String] as parameter with deploymentComment information. Added abstract class Comment in listener-api. 
 
 ### Other changes
 * [#3031](https://github.com/TouK/nussknacker/pull/3031) Attachments are now stored in database. As this feature was rarely used, automatic migration of attachments from disk to db is not provided. To stay consistent db table `process_attachments` had to be truncated.
-  
+* [#2992](https://github.com/TouK/nussknacker/pull/2992) UpdateProcessCommand now uses UpdateProcessComment instead of String as parameter with comment information. Because of that added serialization for UpdateProcessComment. 
```

Comment:
But it serializes to the same format? If so, remove this comment

Reply:
removed it